.github/actions/download-tar-artifact/action.yml

```diff
@@ -10,7 +10,7 @@ inputs:
 runs:
   using: 'composite'
   steps:
-    - name: ⬇️ Download build
+    - name: ⬇ Download build
```

Comment:
@nmerget doesn't a dark icon like this one mean that it would melt into the black background color of the log output, compare to e.g. https://github.com/db-ui/elements/runs/7413690508?check_suite_focus=true ?

Reply:
@mfranzke I guess it's the same: https://github.com/db-ui/elements/runs/7433164447?check_suite_focus=true